### PR TITLE
Adds openapi.yaml file for Swagger.io

### PR DESCRIPTION
**Pull request by jwicks31**
### Description of Changes

Adds a swagger openapi.yaml file. 

- To test load file in [swagger](http://editor.swagger.io/#)

### PR Review Checklist

* [ ] **Have you tested the changes?**
* [ ] **Is QE review required? If so, has this been reviewed by QE?** (For all new features, review by QE is required. Minor changes which already have passing functional tests may not need a new QE review -- when in doubt, loop in a QE reviewer).
* [ ] **Are there code changes that require new unit tests? If so, are those tests included?** (A separate PR for tests is not appropriate).
* [ ] **Are documentation updates required? If so, does this PR include it?** (Separate documentation PR is generally not appropriate).
* [ ] **Are the added/edited files named appropriately and located correctly in the project directory structure?**

### CODE REVIEW

**Comment by jwicks31** [comment]
The only thing that isn't working right now is the example for posting a work. It returns the error `Claim's type must be Work, not string`. I have tried using both 
```yaml
type: 
  Work: "Work"
```
and
```yaml
type: "Work"
```

**Comment by warrenv** [comment]
This is a great start @jwicks31! 
How about replacing the boilerplate in the summary and descriptions with actual text?

File: openapi.yaml [hunk 1]
```diff
@@ -0,0 +1,149 @@
+openapi: 3.0.1
+info:
+  title: Poet Node API
```
Comment (lautarodragan) [inline openapi.yaml]: We should always use "Po.et" wherever possible.

File: openapi.yaml [hunk 2]
```diff
@@ -0,0 +1,149 @@
...
+        code:
+          type: string
+        message:
+          type: string
+      required:
+        - code
+        - message
+    Timestamp:
+      type: 'object'
+      properties:
```
Comment (lautarodragan) [inline openapi.yaml]: Timestamp has a few other properties. Look at the [timestamp interface declaration](https://github.com/poetapp/poet-js/blob/master/src/Interfaces.ts#L55) for reference, and technical tab of [this work](https://explorer.po.et/works/c481118d5ac0527fca2023d3650744c723bb742fcb5977871df4a3d74d0bc1cb) for an example.

Comment (jwicks31) [reply openapi.yaml]: I went ahead and added these in the current file.

**Pull request merged**